lib/yard/code_objects/base.rb

```diff
@@ -132,7 +132,7 @@ def push(value)
     # @see NamespaceMapper.register_separator
     class Base
       # The files the object was defined in. To add a file, use {#add_file}.
-      # @return [Array<String>] a list of files
+      # @return [Array<Array<String, Integer>>] a list of files
```

Comment:
You can use `Array<Array(String, Integer)>` to denote ordered tuples. Try it out at https://yardoc.org/types.html

Reply:
TIL! Thanks, fixed 😄

Reply:
Given `X<A, B>` means `X parameterized by either A or B`, how do you annotate `X parameterized by both A and B` ?